src/dnx.coreclr/dnx.coreclr.cpp

```diff
@@ -242,8 +243,30 @@ HRESULT ExecuteMain(ICLRRuntimeHost2* pCLRRuntimeHost, PCALL_APPLICATION_MAIN_DA
         return hr;
     }
 
+    auto windows_version = dnx::utils::get_windows_version();
+    if (!windows_version)
+    {
+        trace_writer.write(L"Unexpected windows version", false);
```

Comment:
Could we proceed in this case? We could simply use "win-x86" (etc.) as the RID...